### PR TITLE
add Zentral remote TLS server info

### PR DESCRIPTION
**Pull request by headmin**
finally add some info here about Zentral open source project available as remote TLS server for osquery

### CODE REVIEW

**Comment by osqueryer** [comment]
Can one of the osquery admins reply with "ok to test" to kick off a build...

**Comment by osqueryer** [comment]
Can one of the osquery admins reply with "ok to test" to kick off a build...

**Comment by osqueryer** [comment]
Can one of the osquery admins reply with "ok to test" to kick off a build...

File: docs/wiki/deployment/remote.md [hunk 1]
```diff
@@ -180,6 +180,8 @@ Heroku maintains a great project called [Windmill](https://github.com/heroku/win
 
 [Doorman](https://github.com/mwielgoszewski/doorman) is another project that implements the TLS remote settings API. Doorman uses "tags", which can be applied to nodes, packs, and queries, in order to dynamically generate configurations for a unique set or all nodes being managed. Doorman also supports the distributed read and write API, allowing an administrator to schedule ad-hoc queries to be run immediately or in the future.
 
+[Zentral](https://github.com/zentralopensource/zentral) is a Framework and Django based WebServer which implements the TLS remote settings API. Zentral will dynamically generate configurations in "probes", these can contain packs and single queries. Probes process events with input filters (added, removed, "tags", etc.), optionally trigger output actions, and notifications for results returne. Scope for "probes" is available for all devices, business units, and "tagged" device groups, sharding for "probes" is available for scopes. Zentral also supports the distributed read and write API to schedule ad-hoc queries.
```
Comment (theopolis) [inline docs/wiki/deployment/remote.md]: 1. "based web server, which" (comma before which and web server does not need caps).
2. "for results returned" (missing a d)

after those changes: :+1:

Comment (headmin) [inline docs/wiki/deployment/remote.md]: @theopolis done

**Pull request merged**